UI/obs-app.cpp

```diff
@@ -2403,6 +2415,10 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 			blog(LOG_WARNING, "================================");
 			blog(LOG_WARNING, "User is now running multiple "
 					  "instances of OBS!");
+			/* Clear unclean_shutdown flag as multiple instances
+			 * running from the same config will lead to a
+			 * false-positive detection.*/
+			unclean_shutdown = false;
```

Comment:
Are there still valid use cases for multiple instances? We should either fully support it, or remove support for it, but not this weird in-between state, where users can choose to put the application in a state where we cannot ensure full functionality.